java-client/src/main/java/org/scassandra/http/client/PrimingClient.java

```diff
@@ -205,7 +206,7 @@ private String httpGet(String url) {
                 LOGGER.info(errorMessage);
                 throw new PrimeFailedException(errorMessage);
             }
-            LOGGER.debug("Received response from scassandra {}", responseAsString);
+            LOGGER.info("Received response from scassandra {}", responseAsString);
```

Comment:
Should change this back to info otherwise will make info level more noisy

Reply:
Oops, thanks